analysed-packages/libedit/version-20210910-3.1/libedit-20210910-3.1-SPDX2TV.spdx

```diff
@@ -0,0 +1,2711 @@
+SPDXVersion: SPDX-2.3
+DataLicense: CC0-1.0
+
+##-------------------------
+## Document Information
+##-------------------------
+
+DocumentNamespace: http://fossology-6674c79fbf-2qzn2/repo/SPDX2TV_libedit-20210910-3.1.tar.gz_1724853785.spdx
+DocumentName: /srv/fossology/repository/report
+SPDXID: SPDXRef-DOCUMENT
+
+##-------------------------
+## Creation Information
+##-------------------------
+
+Creator: Tool: fossology-4.4.0
+Creator: Person: Sascha Dierberg
+CreatorComment: <text>
+This document was created using license information and a generator from Fossology.
+It contains the license and copyright analysis of libedit 20210910-3.1.
+Please check "LicenseComments" for explanations of concluded licenses.
+</text>
+Created: 2024-08-28T14:03:06Z
+LicenseListVersion: 3.22
+
+##-------------------------
+## Package Information
+##-------------------------
+
+
+PackageName: libedit-20210910-3.1
+PackageFileName: libedit-20210910-3.1.tar.gz
+SPDXID: SPDXRef-upload490
+PackageDownloadLocation: NOASSERTION
+FilesAnalyzed: true
+PackageVerificationCode: eb99086814e11fda5b202c8b48dcac3081af7924
+PackageChecksum: SHA1: 855747cff17110fb59703b5b957d11c8f76b91ef
+PackageChecksum: SHA256: 6792a6a992050762edcca28ff3318cdb7de37dccf7bc30db59fcd7017eed13c5
+PackageChecksum: MD5: 648946c97eb92daed5ca0d08bbd5e6a2
+PackageLicenseConcluded: NOASSERTION
+PackageLicenseDeclared: LicenseRef-BSD-3-Clause
```

Comment:
This is not the correct license, the correct license is LicenseRef-BSD-3-Clause-5982ba0d4fafb162c64f096b24b771a7